k8s/ansible/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 # install system utilities
 RUN apt-get update && apt-get install -y \
     python-pip apt-transport-https bash curl tar openssh-client \
-    sshpass git ca-certificates python-pip apt-utils locales && \
+    sshpass git ca-certificates python-pip apt-utils locales jq && \
```

Comment:
as I remember there are some issues with jq version and cross dependencies with other components.
Could you please pin the version like in pipeline.Dockerfile for example jq=1.5+dfsg-1.3 xvfb=2:1.19.2-1+deb9u5 (Debian distr)

Reply:
I can't because the base image is ubuntu. It doesn't have the specific version of jq.